rfcs/text/0008-threat-intel.md

```diff
@@ -28,21 +28,18 @@ Field | Type | Example | Description
 threat.indicator.first_seen | date | 2020-12-01 | The date and time when intelligence source first reported sighting this indicator
 threat.indicator.last_seen | date | 2020-12-02| The date and time when intelligence source last reported sighting this indicator.
 threat.indicator.sightings | long | 20 | Number of times this indicator was observed conducting threat activity
-threat.indicator.type | keyword | ipv4-addr, domain-name, email-addr | Type of indicator as represented by Cyber Observable in STIX 2.0
-threat.indicator.description | wildcard | 201.10.10.90 was seen delivering Angler EK | Describes the type of action conducted by the threat
-threat.indicator.dataset | keyword | theatintel | Identifies the name of specific dataset from the intelligence source.
-threat.indicator.module | keyword | threatintel.{abusemalware,abuseurl,misp,otx,limo} | Identifies the name of specific module where the data is coming from.
+threat.indicator.type | keyword | ipv4-addr | Type of indicator as represented by Cyber Observable in STIX 2.0
+threat.indicator.description | keyword | 201.10.10.90 was seen delivering Angler EK | Describes the type of action conducted by the threat
+threat.indicator.dataset | keyword | threatintel.{abusemalware,abuseurl,misp,otx,limo} | Identifies the name of specific dataset from the intelligence source.
```

Comment:
@peasead is this one (and some others, I think) redundant with e.g. `event.dataset`, or does this serve a separate purpose?

Reply:
Removed this. Thanks.